src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -456,4 +470,76 @@ tnsr::a<DataType, SpatialDim, Frame> spacetime_deriv_of_norm_of_shift(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
     const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept;
 // @}
+
+namespace Tags {
+/*!
+ * \brief Compute item to get spatial derivatives of the spatial metric from
+ *        the generalized harmonic spatial derivative variable.
+ *
+ * \details See `deriv_spatial_metric()`. Can be retrieved using
+ * `gr::Tags::SpatialMetric` wrapped in `Tags::deriv`.
```

Comment:
`Tags::deriv` doesn't seem to be linking in the doxygen. I'm not sure precisely why, but I'd try fiddling around with how the namespaces is specified.
(I think the same will go for below)

Reply:
I tried fwd declaring `deriv`, explicit scope resolution, etc, but I'm unable to get doxygen to link correctly to `Tags::deriv`. This seems to be a common problem apparently, for e.g. the following dox pages:
 - https://spectre-code.org/structtags_1_1derivcompute
 - https://spectre-code.org/group__numericalalgorithmsgroup#ga94bdfb83e587caae2e8084cd5713b3b8
 - https://spectre-code.org/group__numericalalgorithmsgroup#ga0541b93f207564ad32e49a1ede7a2ee2

also are unable to link to `deriv`. I'm not sure how to fix this. Any suggestions @nilsdeppe @wthrowe ?

Reply:
I would try linking `::Tags::deriv`. What I think is happening is that Doxygen is looking for `deriv` in the local `Tags` namespace, e.g. `gh::Tags` or `gr::Tags`, where there is no `deriv` tag.

Reply:
`::Tags::deriv` is the first thing I tried (and is what I mean by explicit scope resolution in my previous comment). That doesn't help.

Reply:
Huh. this does appear to be a finicky problem. I'm not sure precisely what the doxygen problem here is.
I found an ugly solution which seems to produce the desired format and link:
`<code>@ref deriv "Tags::deriv"</code>`
Though I'd certainly invite any less ugly solutions.

Reply:
Thanks @moxcodes . With an explicit `\ref` it links correctly now.

Reply:
The explicit link was going to be my next suggestion. I agree it's ugly, but I vaguely remember encountering an issue like this before :)